.github/workflows/semgrep-rules-test-historical.yml

```diff
@@ -16,15 +16,26 @@ jobs:
     - uses: actions/checkout@v2
       with:
         path: semgrep-rules
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.9.2
     # need to delete these directories or else we get linter errors during 
     # the "validate" step.
+    - name: install semgrep
+      run: pip3 install semgrep
     - name: delete stats directory
       run: rm -rf semgrep-rules/stats
     - name: delete fingerprints directory
       run: rm -rf semgrep-rules/fingerprints
-    - name: validate rules on semgrep v0.87.0
+    - name: grab historical semgrep version
+      run: |
+        SEMGREP_VERSION=`semgrep --version`
+        SEMGREP_VERSION_SHORT=${SEMGREP_VERSION:2:2}
+        SEMGREP_OLD_VERSION="0.$(($SEMGREP_VERSION_SHORT-3)).0"
+        echo "SEMGREP_OLD_VERSION=$SEMGREP_OLD_VERSION" >> $GITHUB_ENV
+    - name: validate rules on historical semgrep version
       run: |
-        docker run --rm -v ${GITHUB_WORKSPACE}/semgrep-rules:/src returntocorp/semgrep:0.87.0 semgrep --validate --config /src
-    - name: test with semgrep v0.87.0 
+        docker run --rm -v ${GITHUB_WORKSPACE}/semgrep-rules:/src returntocorp/semgrep:${SEMGREP_OLD_VERSION} semgrep semgrep --validate --config /src
```

Comment:
Is this `semgrep semgrep` intentional?

Reply:
It was, I was getting a deprecation error with semgrep, but I don't think that's the actual problem 😓  I'll remove it!